src/selectors/entities/posts.js

```diff
@@ -32,11 +32,22 @@ export function makeGetReactionsForPost() {
         getReactionsForPosts,
         (state, postId) => postId,
         (reactions, postId) => {
-            return Object.values(reactions[postId] || {});
+            return Object.values(reactions[postId] || []);
```

Comment:
Corrected this since it returns an array of object/s.

Reply:
If that's the case, do we even need the `Object.values` here?

Reply:
Hmm, my bad, it's not correct.  I'll revert the change.